### PR TITLE
Khp3 4791

### PR DESCRIPTION
**Pull request by ojwanganto**
## Requirements

- [ ] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary

This PR addresses the following:
1. Adds HTS clinical views - screening and testing
2. Refines utility methods for presenting obs labels based on json form schema question labels
3. Refines MCH clinical views to use the changes in 2 above

## Screenshots
MCH - labor and delivery view showing overridden labels
image

[Screencast from 12-20-2023 12:49:54 AM.webm](https://github.com/palladiumkenya/kenyaemr-esm-3.x/assets/2161877/e5490751-c7b9-40e5-adcc-f4daa2fbe211)

## Related Issue

*None.*

## Other

*None.*

### CODE REVIEW

File: packages/esm-patient-clinical-view-app/src/esm-hts-app/views/hiv-testing/hiv-testing-constants.ts [hunk 1]
```diff
@@ -0,0 +1,247 @@
+export const hivTestingConceptMap = {
```
Comment (ojwanganto) [inline packages/esm-patient-clinical-view-app/src/esm-hts-app/views/hiv-testing/hiv-testing-constants.ts:1]: This is a sample concept map that is used to override default labels from obs.concept.display which come from the concept dictionary

File: packages/esm-patient-clinical-view-app/src/esm-hts-app/views/hiv-testing/tabs/hiv-screening.component.tsx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+      description={headerTitle}
+      headerTitle={headerTitle}
+      launchOptions={{
+        displayText: t('add', 'Add'),
+        moduleName: 'HTS Clinical View',
+      }}
+      filter={(encounter) => {
+        return encounter.form.uuid == htsScreeningFormUUID;
+      }}
+      formConceptMap={hivScreeningConceptMap}
```
Comment (ojwanganto) [inline packages/esm-patient-clinical-view-app/src/esm-hts-app/views/hiv-testing/tabs/hiv-screening.component.tsx:107]: This is how the concept map is injected

**Comment by donaldkibet** [comment]
@ojwanganto this looks like the same as [this](https://github.com/palladiumkenya/kenyaemr-esm-3.x/pull/118) should we close this PR

**Comment by ojwanganto** [comment]
closing this one.

**Pull request closed without merging**